File: BEE 1036.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        double A = input.nextDouble();
        double B = input.nextDouble();
        double C = input.nextDouble();
        double root = (B * B - 4 * A * C);
        if(A != 0 && root > 0){
            double R1 = (- B + Math.sqrt(root)) / (2 * A);
            double R2 = (- B - Math.sqrt(root)) / (2 * A);
            System.out.printf("R1 = %.5f\n", R1);
            System.out.printf("R2 = %.5f\n", R2);
        }
        else{
            System.out.println("Impossivel calcular");
        }
    }
}

File: BEE 1036.php
<?php

    list($a, $b, $c) = explode(" ", readline());
    $a = (float) $a;
    $b = (float) $b;
    $c = (float) $c;
    $d = ($b * $b) - (4 * $a * $c);
    if (($d > 0) and ($a > 0)){
        $r1 = (- $b + sqrt($d)) / (2 * $a);
        $r2 = (- $b - sqrt($d)) / (2 * $a);
        $r1 = number_format($r1, 5, ".", "");
        $r2 = number_format($r2, 5, ".", "");
        echo "R1 = {$r1}\n";
        echo "R2 = {$r2}\n";
    }
    else{
        echo "Impossivel calcular\n";
    }

?>